.github/workflows/openai.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: |
           docker --version
           python -m pip install --upgrade pip wheel
-          pip install -e .[autogen,blendsearch]
+          pip install -e .[mathchat,blendsearch]
```

Comment:
Can we install [mathchat] only when necessary? I want to make sure installing [autogen] is enough to pass non-mathchat tests. There is one python version which doesn't run any mathchat tests.

Reply:
sure